src/index.js

```diff
@@ -14,15 +14,14 @@ let initModule = false;
  * @param  {string} [config.LOGS_TOKEN] Token used by internal logger to send logs to
  * @return {Object}        The configured providers
  */
-module.exports = function(config) {
-  //if no param is provided, we want to return the cached modules
+module.exports = (config) => {
+  // if no param is provided, we want to return the cached modules
   if (!config) {
```

Comment:
Empty object can be a valid initializer (like to use the module only with logger and not bugnsag nor logentries) so it should not enter the if and init the module